src/test/java/com/smartcar/sdk/SmartcarExceptionTest.java

```diff
@@ -138,19 +138,4 @@ public void testSmartcarExceptionFactoryWithInvalidJsonBody() throws Exception {
 
     Assert.assertEquals(ex.getMessage(), response);
   }
-
-  /**
-   * Test SmartcarException.Factory raising generic exception
-   */
-  @Test
-  public void testSmartcarExceptionFactoryWithGenericException() throws Exception {
```

Comment:
So I had written this test assuming there could be an exception in the function call chain `response.body().string()` however , that was never the bug. The bug was because of the result of this being `""` (empty string) . So i find it ok to remove this test case. Let me know if this sounds ok.